qiskit/transpiler/passes/routing/algorithms/bip_model.py

```diff
@@ -125,6 +125,9 @@ def __init__(self, dag, coupling_map, qubit_subset, dummy_timesteps=None):
             self.gates.extend([[]] * dummy_timesteps)
 
         self.bprop = None  # Backend properties to compute cx fidelities (set later if necessary)
+        self.default_cx_error_rate = (
+            5e-3  # Default cx error rate in case backend properties are not available
+        )
```

Comment:
`self.default_cx_error_rate = None` would be better here (to avoid to have a magic value 5e-3 in two places).

Reply:
Done.